src/selectors/cpu.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import { createSelector } from 'reselect';
+
+import { getThreads, getSampleUnits, getMeta } from './profile';
+
+import type { Selector } from 'firefox-profiler/types';
+
+export const getIsCPUUtilizationProvided: Selector<boolean> = createSelector(
+  getSampleUnits,
+  getMeta,
+  getCPUProcessedThreads,
+  (sampleUnits, meta, threads) => {
+    return (
+      sampleUnits !== undefined &&
+      // Currently checking the features array for 'cpu' feature should be enough,
+      // but in the future we may remove that feature and enable it perminantly.
+      // Therefore we are also checking the samples table to see if we have CPU
+      // delta values.
+      ((meta.configuration && meta.configuration.features.includes('cpu')) ||
+        threads.some(thread => thread.samples.threadCPUDelta !== undefined))
```

Comment:
It works when we don't have this information because `cpu` element will not be in the `meta.configuration.features` array. See `meta.configuration.features.includes('cpu'))` above.